OpenSim/Tests/testIterators/testIterators.cpp

```diff
@@ -150,7 +150,11 @@ int main()
         // to recompose existing components, this will need continual updating. For example,
         // Joint's often add PhysicalOffsetFrames to handle what used to be baked in location
         // and orientation offsets.
-        ASSERT(numComponents == 77); 
+
+        // Updated below from 77 to 65 when pennation and activation models in
+        // Thelen2003Muscle were converted into private (non-serialized)
+        // subcomponents (2 subcomponents x 6 muscles).
+        ASSERT(numComponents == 65); 
```

Comment:
Hmm I didn't expect this. I thought we would still want to iterate over these subcomponents; that they were just hidden in XML. I now feel that we should *not* include these subcomponents when iterating, but this causes some inconsistency. I imagine that users would still want to be able to access the state variables from these subcomponents (as `/soleus_r/activation_model/activation` or something like that), which would make the class visible in some scenarios, but not visible in others.